spec/features/idv/analytics_spec.rb

```diff
@@ -71,6 +71,10 @@
       jurisdiction_in_maintenance_window: false }
   end
 
+  let(:state_id_resolution_with_id_type) do
+    state_id_resolution.merge(state_id_type: 'drivers_license')
```

Comment:
question (non-blocking): do we ever expect `state_id_type` to be more than one type (e.g., state ID cards for non-drivers) or to not even be a state-issued ID?

Reply:
I think what I'm actually asking is do we expect the type of ID to change the shape of the event data and if so, can we account for that in the specs?

Reply:
Good question! Right now the only two values encountered are 'drivers_license' and 'state_id_card'. For how we handle proofing, they contain the same attributes other than the ID type.